tests/test_environment.py

```diff
@@ -101,78 +179,121 @@ def test_noaa_ruc_sounding_atmosphere(mock_show, example_env):
 @pytest.mark.slow
 @patch("matplotlib.pyplot.show")
 def test_gfs_atmosphere(mock_show, example_env_robust):
+    """Tests the Forecast model with the GFS file. It does not test the values,
+    instead the test checks if the method runs without errors.
+
+    Parameters
+    ----------
+    mock_show : mock
+        Mock object to replace matplotlib.pyplot.show() method.
+    example_env_robust : rocketpy.Environment
+        Example environment object to be tested.
+    """
     example_env_robust.set_atmospheric_model(type="Forecast", file="GFS")
     assert example_env_robust.all_info() == None
 
 
 @pytest.mark.slow
 @patch("matplotlib.pyplot.show")
 def test_nam_atmosphere(mock_show, example_env_robust):
+    """Tests the Forecast model with the NAM file.
+
+    Parameters
+    ----------
+    mock_show : mock
+        Mock object to replace matplotlib.pyplot.show() method.
+    example_env_robust : rocketpy.Environment
+        Example environment object to be tested.
+    """
     example_env_robust.set_atmospheric_model(type="Forecast", file="NAM")
     assert example_env_robust.all_info() == None
 
 
 # Deactivated since it is hard to figure out and appropriate date to use RAP forecast
-# @pytest.mark.slow
-# @patch("matplotlib.pyplot.show")
-# def test_rap_atmosphere(mock_show, example_env_robust):
-#     today = datetime.date.today()
-#     example_env_robust.set_date((today.year, today.month, today.day, 8)) # Hour given in UTC time
-#     example_env_robust.set_atmospheric_model(type='Forecast', file='RAP')
-#     assert example_env_robust.all_info() == None
+@pytest.mark.skip(reason="legacy tests")
+@pytest.mark.slow
+@patch("matplotlib.pyplot.show")
+def test_rap_atmosphere(mock_show, example_env_robust):
+    today = datetime.date.today()
+    example_env_robust.set_date((today.year, today.month, today.day, 8))
+    example_env_robust.set_atmospheric_model(type="Forecast", file="RAP")
+    assert example_env_robust.all_info() == None
```

Comment:
Please take a look at the `@pytest.mark.skip(reason="legacy tests")` decorator. 
The test is still deactivated, it will not run under any circuntancy.
I am not sure if we can think in a solution to the problem regarding dates in this test.

I was not confortable with the inconsistent way to deactivate tests. Some were commented, some were deleted and some were marked as "legacy tests". Now everything is simply marked as legacy, easier to maintain.